README.md

```diff
@@ -195,7 +195,7 @@ The `@uswds/uswds` module is now installed as a dependency. You can use the comp
 ### Install the package directly from GitHub
 If you’re using a framework or package manager that doesn’t support npm, you can find the source files in this repository and use them in your project. Otherwise, we recommend that you follow the steps outlined in this section.
 
-1. Download the [USWDS package](https://github.com/uswds/uswds/releases/download/v3.7.1/uswds-uswds-3.7.1.tgz) directly from the latest USWDS release and uncompress that file.
+1. Download the [USWDS package](https://github.com/uswds/uswds/releases/download/v3.8.0/uswds-uswds-3.8.0.tgz) directly from the latest USWDS release and uncompress that file.
```

Comment:
> [!TIP]
> We should consider changing this to point to `https://github.com/uswds/uswds/releases` since it displays latest first.

Reply:
I think we should do that. This is a pain to maintain and doesn't give much added benefit. In fact, a link triggering a download is probably unexpected behavior here.